pkg/controller/framework/informers/core.go

```diff
@@ -201,3 +205,94 @@ func (f *pvInformer) Lister() *cache.StoreToPVFetcher {
 	informer := f.Informer()
 	return &cache.StoreToPVFetcher{Store: informer.GetStore()}
 }
+
+// NewPodInformer returns a SharedIndexInformer that lists and watches all pods
```

Comment:
These are straight moves until I'm able to fixup all our other uses.  In openshift we drive our tests from fake clients feeding an informer factory.  We should do something similar in kube, but it will take a while.